### PR TITLE
contribution/iadvize

### PR DESCRIPTION
**Pull request by davhamelin**
add connector iadvize to TOCK

### CODE REVIEW

File: bot/connector-iadvize/pom.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>ai.tock</groupId>
+        <artifactId>tock-bot</artifactId>
+        <version>22.3.2-SNAPSHOT</version>
```
Comment (vsct-jburet) [inline bot/connector-iadvize/pom.xml:26]: 22.3.3-SNAPSHOT now

File: bot/connector-iadvize/src/main/kotlin/IadvizeBuilder.kt [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+ * limitations under the License.
+ */
+
+package ai.tock.bot.connector.iadvize
+
+import ai.tock.bot.connector.ConnectorType
+import ai.tock.bot.connector.iadvize.model.response.conversation.QuickReply
+import ai.tock.bot.connector.iadvize.model.response.conversation.payload.TextPayload
+import ai.tock.bot.connector.iadvize.model.response.conversation.Duration
+import ai.tock.bot.connector.iadvize.model.response.conversation.reply.*
```
Comment (vsct-jburet) [inline bot/connector-iadvize/src/main/kotlin/IadvizeBuilder.kt:24]: avoid * imports

File: bot/connector-iadvize/src/main/kotlin/IadvizeConnector.kt [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+ */
+
+package ai.tock.bot.connector.iadvize
+
+import ai.tock.bot.connector.ConnectorBase
+import ai.tock.bot.connector.ConnectorCallback
+import ai.tock.bot.connector.ConnectorData
+import ai.tock.bot.connector.ConnectorMessage
+import ai.tock.bot.connector.iadvize.model.DistributionRule
+import ai.tock.bot.connector.iadvize.model.request.*
```
Comment (vsct-jburet) [inline bot/connector-iadvize/src/main/kotlin/IadvizeConnector.kt:24]: * import

File: bot/connector-iadvize/pom.xml (hunk 1 above)
Comment (davhamelin) [reply bot/connector-iadvize/pom.xml]: changed

File: bot/connector-iadvize/src/main/kotlin/IadvizeBuilder.kt (hunk 2 above)
Comment (davhamelin) [reply bot/connector-iadvize/src/main/kotlin/IadvizeBuilder.kt]: ok (automatic change by intellij) it's fixed

File: bot/connector-iadvize/src/main/kotlin/IadvizeConnector.kt (hunk 3 above)
Comment (davhamelin) [reply bot/connector-iadvize/src/main/kotlin/IadvizeConnector.kt]: ok (automatic change by intellij) it's fixed

**Review by vsct-jburet — APPROVED** [review]

**Comment by vsct-jburet** [comment]
Thank you !

**Pull request merged**